README.md

```diff
@@ -1,2 +1,16 @@
-# cyberdisc-bot
-An attempted bot for the CyberDiscovery discord server.
+# Cyber Discovery Discord Bot [![Build Status](https://travis-ci.com/CyberDiscovery/cyberdisc-bot.svg?branch=master)](https://travis-ci.com/CyberDiscovery/cyberdisc-bot) [![Discord](https://discordapp.com/api/guilds/409851296116375565/embed.png)](https://discord.gg/AQPh34Y)
+The bot for the Cyber Discovery Community Discord Server!
+
+## Commands
+### General Commands:
+* **`:help`** - Displays information about the usage and syntax of the commands
+
+### Cyber Security Commands:
+* **`:TODO`** - Gets information about a specific CyberStart Game challenge.
+* **`:haveibeenpwned email@exmaple.com`** - Searches haveibeenpwned.com for breached accounts.
+* **`:hasitbeenpwned password`** - Searches pwnedpasswords.com for breached passwords.
+
+### Commands for Fun:
+* **`:lmgtfy Stupid Question?`** - Returns a LMGTFY URL for the given question.
+* **`:react OMG`** - Reacts to the previous message with an emoji version of the given string.
```

Comment:
The react command currently doesn't work like this, it just takes space separated emojis. This is on the to-do list to be reimplemented.

Reply:
Please write that in